### ISSUE TITLE
[Suggestion] Hope to Add Instructions for BE Configurations

### ISSUE THREAD

**Issue by hxianshun** [apache/incubator-doris#3740]
https://github.com/apache/incubator-doris/blob/master/docs/zh-CN/administrator-guide/config/be_config.md

In this page, most of configurations have no instructions. Users can't know what the configuration mean and how the configuration use, so I suggest add instructions for these configurations

**Comment by HangyuanLiu** [comment]
We are gradually improving the documentation. The detailed config is commented and explained in the config.cpp/Config.java
 If you have a certain configuration explanation, welcome to contribute code.

**Comment by acelyc111** [comment]
For BE configs, how about to add a new description parameter for config defination like gflag style, for example:
```
    CONF_Int32(be_port, "9060", "port on which ImpalaInternalService is exported");
```
And then, we can dump all configs to a file (markdown format?), we can use this file directly in documentation.

**Comment by hxianshun** [comment]
> For BE configs, how about to add a new description parameter for config defination like gflag style, for example:
> 
> ```
>     CONF_Int32(be_port, "9060", "port on which ImpalaInternalService is exported");
> ```
> 
> And then, we can dump all configs to a file (markdown format?), we can use this file directly in documentation.

Good idea, but the translation of other languages how to show? I think not only description or translation needed, maybe explain how to use these confis to make doris achieve high performance is more important.

**Comment by kangkaisen** [comment]
> For BE configs, how about to add a new description parameter for config defination like gflag style, for example:
> 
> ```
>     CONF_Int32(be_port, "9060", "port on which ImpalaInternalService is exported");
> ```
> 
> And then, we can dump all configs to a file (markdown format?), we can use this file directly in documentation, no need to maintenance code and doc separatly.

+1. We could refer to Apache Arrow. https://arrow.apache.org/docs/cpp/  It's doc was generated by code comment.
